website/scaleway.erb

```diff
@@ -124,6 +124,15 @@
           </ul>
         </li>
 
+        <li>
```

Comment:
- Move it between `Baremetal Resources (Beta)` and `Kubernetes Resources (Beta)`.